src/VisualStudio/Core/Def/ID.RoslynCommands.cs

```diff
@@ -19,13 +19,14 @@ public static class RoslynCommands
             public const int ReferencesContextAddAnalyzer = 0x010d;
             public const int DiagnosticContextMenu = 0x010e;
             public const int DiagnosticSeverityGroup = 0x010f;
-            public const int SetSeverityError = 0x0110;
-            public const int SetSeverityWarning = 0x0111;
-            public const int SetSeverityInfo = 0x0112;
-            public const int SetSeverityHidden = 0x0113;
-            public const int SetSeverityNone = 0x0114;
-            public const int OpenDiagnosticHelpLink = 0x0116;
-            public const int SetActiveRuleSet = 0x0118;
+            public const int SetSeverityDefault = 0x0110;
+            public const int SetSeverityError = 0x0111;
+            public const int SetSeverityWarning = 0x0112;
+            public const int SetSeverityInfo = 0x0113;
+            public const int SetSeverityHidden = 0x0114;
+            public const int SetSeverityNone = 0x0115;
+            public const int OpenDiagnosticHelpLink = 0x0117;
+            public const int SetActiveRuleSet = 0x0119;
         }
```

Comment:
There's no need to do this, and since these need to be declared in two places and kept in sync (here and in the .vsct file) it's better to just add new commands to the end rather than updating them all to try and keep them grouped.